### ISSUE TITLE
Assertion failed: update should only be necessary if node already has focus

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#201]
This occurred when I first added the Voicing mixin to BookNode. 

<details>

```diff
Index: friction/js/friction/view/book/BookNode.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/friction/js/friction/view/book/BookNode.js b/friction/js/friction/view/book/BookNode.js
--- a/friction/js/friction/view/book/BookNode.js	(revision 1ecb12e5930aa64c719f88acf58d7ba6e1e419fd)
+++ b/friction/js/friction/view/book/BookNode.js	(date 1626197590369)
@@ -12,6 +12,7 @@
 import Shape from '../../../../../kite/js/Shape.js';
 import merge from '../../../../../phet-core/js/merge.js';
 import FocusHighlightPath from '../../../../../scenery/js/accessibility/FocusHighlightPath.js';
+import Voicing from '../../../../../scenery/js/accessibility/voicing/Voicing.js';
 import Node from '../../../../../scenery/js/nodes/Node.js';
 import SoundClip from '../../../../../tambo/js/sound-generators/SoundClip.js';
 import soundManager from '../../../../../tambo/js/soundManager.js';
@@ -63,6 +64,8 @@
     // init drag and a11y options for the draggable book
     if ( options.drag ) {
 
+      this.initializeVoicing( options );
+
       // instrument this book, but not the other
       options.tandem = tandem;
 
@@ -170,6 +173,8 @@
   }
 }
 
+Voicing.compose( Node );
+
 friction.register( 'BookNode', BookNode );
 
 export default BookNode;
\ No newline at end of file
Index: joist/js/preferences/PreferencesProperties.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/joist/js/preferences/PreferencesProperties.js b/joist/js/preferences/PreferencesProperties.js
--- a/joist/js/preferences/PreferencesProperties.js	(revision 28a10c682eef20bd0a391eaec610e11999d3de07)
+++ b/joist/js/preferences/PreferencesProperties.js	(date 1626196575964)
@@ -19,7 +19,7 @@
     // @public {BooleanProperty} - Whether or not "Interactive Highlights" are enabled for the simulation. If enabled,
     // focus highlights will appear around focusable components with 'over' events, and persist around the focused
     // element even with mouse and touch interaction.
-    this.interactiveHighlightsEnabledProperty = new BooleanProperty( false );
+    this.interactiveHighlightsEnabledProperty = new BooleanProperty( true );
 
     // @public {BooleanProperty} - Whether or not "Gesture Controls" are enabled for the simulation. If enabled,
     // touch screen input will change to work like a screen reader. Horizontal swipes across the screen will move focus,

```

</details>

1. Turn on voicing (already done by `?voicingInitiallyEnabled` )
2. Turn on interactive highlights (already done by patch)
3. click on the macro book node.

I'm not really sure what focus means for a Node these days. `this.node.focused` is what should be true but is false. Isn't this property tied to PDOM focus? Do we have anything for "any focus"? If this is supposed to be for any focus, should it be moved from ParallelDOM to Node proper?

**Comment by jessegreenberg** [comment]
I think this is a subset issue of #200. The work being done in `HighlightOverlay.onFocusHighlightChange`, is specific to the PDOM highlight. In #200 we will likely add support for other kinds of highlights which will have their own listeners and checks for focus. Lets consider that issue first.

**Comment by zepumph** [comment]
I added a TODO and turned off interactive highlights as an option to preferences until we sort this.

**Comment by zepumph** [comment]
@jessegreenberg and discussed this an we will first work on https://github.com/phetsims/scenery/issues/1230 to get this working.
